### ISSUE TITLE
Dupe Bug with Slotted Books

### ISSUE THREAD

**Issue by Carrots4U** [Nuchaz/BiblioCraft#298]
You are able to dupe any item by placing it into the item slot of a slotted book and pressing "Save" and right clicking once quickly and then pressing "Q". You are able to take the item out of the gui and pick up the book with the item still inside.

**Comment by Nuchaz** [comment]
Hmmm, that's definitely not good. I had fixed another dupe bug with the slotted book awhile back, but this sound a little different. I'll investigate and see what I can do for a fix. It will likely be after the new website is finished, but I definitely want to get back to some BiblioCraft dev as soon as the new website is up.

**Comment by SirWilli** [comment]
Here is another dupe with the slotted book:
https://youtu.be/jemDwT8WFEk

**Comment by Nuchaz** [comment]
Thanks for the video. I am working on trying to solve these various dupe bugs with the slotted book. I never would have thought an item with an inventory would have so many ways it could be exploited. Hopefully I can get it locked down though.

**Comment by SirWilli** [comment]
All backpack mods started with at least one dupe bug :D

**Comment by Nuchaz** [comment]
I can't seem to actually get either one of these dupes to work in SSP. I setup a local SMP server next and try to reproduce. The first one I am wondering if I need a little lag or something to make it work. The second one with the video, I tried using a setup I have that has all those mods, but some of the mods might not be as up to date at the infinity pack, and I couldn't reproduce the issue. I'll try SMP next just to be sure though.

**Comment by Nuchaz** [comment]
Alright, I've been able get them both reliably duplicated and have figured out a fix for both of them. It should hopefully be impossible to dupe anything with the slotted book. I've extended what I learned about the slotted book to the Atlas just to be sure nothing can be duped from that either. I plan to patch up a couple last things and do a little more testing and I will try to get a release out soon. So look for a new version on the website (bibliocraftmod.com) soon and it will include a fix for this.
